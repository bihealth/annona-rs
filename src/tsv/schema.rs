//! Schema inference and representation for TSV files.
