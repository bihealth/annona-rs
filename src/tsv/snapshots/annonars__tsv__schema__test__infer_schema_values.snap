---
source: src/tsv/schema.rs
expression: record
---
FileSchema {
    columns: [
        ColumnSchema {
            name: "a",
            typ: String,
        },
        ColumnSchema {
            name: "b",
            typ: String,
        },
        ColumnSchema {
            name: "c",
            typ: Float,
        },
        ColumnSchema {
            name: "d",
            typ: Integer,
        },
    ],
    null_values: [
        ".",
    ],
}
